corehq/apps/hqwebapp/templates/hqwebapp/includes/global_navigation_bar.html

```diff
@@ -6,7 +6,8 @@
 <ul class="nav navbar-nav" role="menu">
     <li class="dropdown">
         <a href="#" class="dropdown-toggle dropdown-toggle-with-icon track-usage-link"
-           data-toggle="dropdown" data-category="Nav Bar" data-action="Click Gear Icon">
+           data-toggle="dropdown" data-category="Nav Bar" data-action="Click Gear Icon"
+           aria-label="Settings">
             <i class="fa fa-cog nav-main-icon"></i> <span class="responsive-label">{% trans "Manage" %}</span>
```

Comment:
Is there a reason for the mismatch in "Manage" and "Settings"? (I honestly maybe like settings more, but could cause for some confusion)

Reply:
Oh cool I didn't see that-- "Manage Settings" is a great idea

Reply:
Are we finalizing on "Manage Settings" then?